contrib/charts/dragonfly/templates/service.yaml

```diff
@@ -17,6 +17,7 @@ spec:
   {{- if and (eq .Values.service.type "LoadBalancer") (ne .Values.service.loadBalancerIP "") }}
   loadBalancerIP: {{ .Values.service.loadBalancerIP }}
   {{- end }}
+  clusterIP: {{ .Values.service.clusterIP }}
```

Comment:
Can we update this to be toggled only when the type is Cluster and the value is not empty? Keeps it cleaner.

Reply:
Also we can do the same for `hostNetwork` toggle.